frameit/lib/frameit/strings_parser.rb

```diff
@@ -40,7 +40,7 @@ def self.parse(path)
     end
 
     def self.encoding_type(path)
-      `file --mime-encoding #{path}`.downcase
+      `file --mime-encoding "#{path}"`.downcase
```

Comment:
A better way to achieve this in a cross platform way would be to use `.shellescape` on the string: http://ruby-doc.org/stdlib-2.0.0/libdoc/shellwords/rdoc/Shellwords.html#method-c-shellescape This is also how the rest of fastlane makes path shell safe.